theano/gpuarray/dnn.py

```diff
@@ -1422,7 +1422,7 @@ class GpuDnnBatchNorm(DnnBase):
         value is 1e-5 (imposed by cuDNN).
     """
 
-    __props__ = ('mode', 'epsilon')
+    __props__ = ('mode',)
 
     def __init__(self, mode='per-activation', epsilon=1e-4):
```

Comment:
epsilon should not be a param of init anymore, but a param of make_node. Idem for all Ops.